kogito-usertasks-with-security-oidc-keycloak/src/test/java/org/acme/travels/BearerTokenAuthorizationTest.java

```diff
@@ -0,0 +1,131 @@
+package org.acme.travels;
+
+import java.util.Map;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.Test;
+import org.keycloak.representations.AccessTokenResponse;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
+ */
+@QuarkusTest
+public class BearerTokenAuthorizationTest {
```

Comment:
I believe this test should be disabled by default unless we have option to run some kind of embedded Keycloak...

Reply:
@mswiderski that's true, I tried to replicate the quarkus test way but it's true, it supposes that the keycloak is running and well configured. I thought that was something similar that we had with the infinispan service, but seeing the  https://github.com/kiegroup/kogito-runtimes/pull/197 work to ensure the dependent services running, maybe makes sense to remove this test, at least for now.